.github/workflows/release.yaml

```diff
@@ -111,6 +113,34 @@ jobs:
         echo "Hash of package should be $ARTIFACT_HASH."
         echo "$ARTIFACT_HASH" | base64 --decode | sha256sum --strict --check --status || exit 1
 
+    - name: Log in to GitHub Container Registry
+      run: docker login ghcr.io --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }}
+
+    # The Docker image will be built with tags: latest and ${{ RELEASE_TAG }}
+    - name: Build the Docker image
+      run: |
+        RELEASE_TAG=$(git describe --tags --abbrev=0)
+        ./scripts/dev_scripts/build_docker.sh "${IMAGE_NAME}" ${{ github.workspace }} "${RELEASE_TAG}"
```

Comment:
Why not `"$IMAGE_NAME"` and `"$RELEASE_TAG"`?

Reply:
I think this comment is duplicated - https://github.com/oracle-samples/macaron/pull/76#discussion_r1137899352